packages/shared/src/server/repositories/dashboards.ts

```diff
@@ -318,7 +318,7 @@ export const getModelUsageByUser = async (
   const timeFilter = chFilter.find(
     (f) =>
       f.clickhouseTable === "observations" &&
-      f.field === "start_time" &&
+      f.field.includes("start_time") &&
```

Comment:
Field is set to o.start_time. With this approach we only check if something like it is present.